src/main/java/org/ladysnake/blabber/api/client/BlabberDialogueScreen.java

```diff
@@ -230,6 +241,16 @@ public void render(DrawContext context, int mouseX, int mouseY, float tickDelta)
             boolean selected = i == this.selectedChoice;
             int choiceColor = choice.unavailabilityMessage().isPresent() ? lockedChoiceColor : selected ? selectedChoiceColor : this.choiceColor;
             context.drawTextWrapped(this.textRenderer, choice.text(), choiceListMinX, y, choiceListMaxWidth, choiceColor);
+
+            for (String illustrationList : choice.illustrations()) {
+                List<DialogueIllustration> illustrations = this.handler.getIllustrations().get(illustrationList);
```

Comment:
okay hear me out : what if, instead of dealing with lists here, we had a dedicated `composition` illustration type ?

Reply:
That would make writing these more annoying: you'd have to write {"type": "blabber:list" etc every time you want a list. It would make writing single illustration sets easier, but rn you just need to add 2 characters to go from a single illustration to a list.

It would make some of the code more straightforward, but this just means an extra loop here and there, so I'm fine with it.

Reply:
Technically we could make it so an array is automatically parsed as a composition - that's what loot tables do